src/net.h

```diff
@@ -1015,7 +1015,7 @@ class CConnman
 
     // Network stats
     void RecordBytesRecv(uint64_t bytes);
-    void RecordBytesSent(uint64_t bytes);
+    void RecordBytesSent(uint64_t bytes, bool increaseMaxOutbound);
```

Comment:
nit, per doc/developer-notes.md current naming style is snakecase, e.g. `increase_max_outbound`